src/modules/dashboard/components/ExpenditurePage/Streaming/FieldError/FieldError.tsx

```diff
@@ -0,0 +1,24 @@
+import { useField } from 'formik';
+import React from 'react';
+import { useIntl } from 'react-intl';
+import classNames from 'classnames';
+
+import styles from './FieldError.css';
+
+interface Props {
+  name: string;
+}
+
+const FieldError = ({ name }: Props) => {
```

Comment:
I just had a thought (and I might be 100% wrong), is this component needed or could you maybe use the existing `InputStatus`?